bin/prestart.sh

```diff
@@ -47,18 +47,19 @@ else
   # set a nice default permalink structure
   wp --allow-root option update permalink_structure '/%year%/%monthnum%/%postname%/'
   # set theme
-  if [ $WORDPRESS_ACTIVE_THEME ]
+  if [ -n "$WORDPRESS_ACTIVE_THEME" ]
   then
     wp --allow-root theme activate $WORDPRESS_ACTIVE_THEME
   fi
 
-  if [ $WORDPRESS_TEST_DATA ]
+  if [[ $WORDPRESS_TEST_DATA=true ]]
```

Comment:
"true" has no special meaning in bash, so if this works, it's doing a string comparison and won't match "`True`", "`TRUE`", or "`1`" as you'd expect in PHP. Perhaps this should be tested the same as `$WORDPRESS_ACTIVE_THEME` above and many WP constants are, and simply test if it's set with a non-empty value? `[ -n "WORDPRESS_TEST_DATA" ]` should work reliably.